frontend/src/prompted_readings/student_view.js

```diff
@@ -243,15 +341,25 @@ class Study extends React.Component {
     }
 
     handleStartClick() {
+        /*
```

Comment:
This looks very good.  Thank you.  Consider looking up the JSDoc format which uses two *s at the start and precedes the function name (unlike Python docs) -- those docs can automatically be highlighted and extracted.

Reply:
Thanks for the heads up! I'll switch everything over.